src/main/java/greencity/controller/UserOwnSecurityController.java

```diff
@@ -0,0 +1,24 @@
+package greencity.controller;
+
+import greencity.dto.user_own_security.UserRegisterDto;
+import greencity.service.UserOwnSecurityService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/ownSecurity")
+@AllArgsConstructor
+public class UserOwnSecurityController {
+
+    private UserOwnSecurityService service;
+
+    @PostMapping
+    public void register(@Valid @RequestBody UserRegisterDto dto) {
+        service.register(dto);
+    }
+}
```

Comment:
I think it would be better to have a return here. For example, to return a status or a validation error.

Reply:
if we have some errors we throw exception that will be catch by error handler. So if no any errors that mean that user successfully registered. That mean that i need not return anything.

Reply:
https://medium.com/hashmapinc/rest-good-practices-for-api-design-881439796dc9